llvm/lib/Transforms/Utils/SimplifyCFG.cpp

```diff
@@ -6378,18 +6378,20 @@ ShouldBuildLookupTable(SwitchInst *SI, uint64_t TableSize,
 }
 
 static bool ShouldUseSwitchConditionAsTableIndex(
-    ConstantInt &MinCaseVal, const ConstantInt &MaxCaseVal,
+    ConstantInt &BeginCaseVal, const ConstantInt &EndCaseVal,
```

Comment:
```suggestion
    const ConstantInt &BeginCaseVal, const ConstantInt &EndCaseVal,
```
`BeginCaseVal` is read-only.